renderers/network.py

```diff
@@ -0,0 +1,29 @@
+from rgbmatrix import graphics
+from utils import center_text_position
+import data.layout
+import debug
+
+NETWORK_ERROR_TEXT = "!"
+
+class NetworkErrorRenderer:
+  def __init__(self, canvas, data):
+    self.canvas = canvas
+    self.data   = data
+    self.layout = data.config.layout
+    self.colors = data.config.scoreboard_colors
+
+  def render(self):
+    if self.data.network_issues == True:
+      font = self.layout.font("network")
+      coords = self.layout.coords("network")
+      bg_coords = coords["background"]
+      text_color = self.colors.graphics_color("network.text")
+      bg_color = self.colors.color("network.background")
+
+      # Fill in the background so it's clearly visible
+      for x in range(bg_coords["width"]):

```

Comment:
can't we use canvas.Fill here? Not sure the difference in performance

Reply:
canvas.Fill just fills the entire canvas with a color. The indicator is just a small square in the corner and is customizable in the coords so I wouldn't really want to fill the entire screen.